ext/js/display/display-anki.js

```diff
@@ -1303,6 +1326,34 @@ export class DisplayAnki {
 
         return 'linear-gradient(to right,' + gradientSlices.join(',') + ')';
     }
+
+    /**
+     * Get the highest priority state from a list of Anki queue states.
+     *
+     * Priority order:
+     *   - -3, -2 → "buried"
+     *   - -1 → "suspended"
+     *   -  2 → "review"
+     *   -  1, 3 → "learning"
+     *   -  0 → "new" (default fallback)
```

Comment:
It's worth mentioning the source of this (https://github.com/ankidroid/Anki-Android/wiki/Database-Structure).